connection.go

```diff
@@ -255,6 +259,12 @@ func (b *ConnectionBuilder) Insecure(flag bool) *ConnectionBuilder {
 	return b
 }
 
+// Transport allows setting a transport layer into the connection for capturing and manipulating the request or response.
+func (b *ConnectionBuilder) Transport(transport WrappedTransport) *ConnectionBuilder {
```

Comment:
Rename this to `TransportWrapper`.

Make sure that lines are 100 caracters or less.

Reply:
Fixed